multi-agent/adapters/outbound/mongo/admin_config_reader.py

```
@@ -0,0 +1,51 @@
+class MongoAdminConfigReader(AdminConfigReaderPort):
+    """Read-only reader for the centralized admin config collection.
+
+    Args:
+        mongodb_ip: MongoDB host.
+        mongodb_port: MongoDB port.
+        db_name: Database that holds the ``admin_config`` collection
+                 (defaults to ``"config"`` — the backend's database).
+        coll_name: Collection name (defaults to ``"admin_config"``).
+    """
+
+    def __init__(
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)